nextflow_schema.json

```diff
@@ -89,6 +89,27 @@
                 }
             }
         },
+        "databases": {
+            "title": "Databases",
+            "type": "object",
+            "description": "The optional databases of profiles and cluster addresses that can be used.",
+            "fa_icon": "fas fa-terminal",
+            "default": "",
+            "properties": {
+                "db_profiles": {
+                    "type": "string",
+                    "pattern": "^\\S+\\.tsv$",
```

Comment:
I am wondering if you could change this to allow both uncompressed `.tsv` and compressed `.tsv.gz` files as input? Since I would like to store the database files as gzipped tsv.

I know this will require a few additional changes outside of this file so that concatenation works properly (does csvtk work with gzipped files as well?)

Reply:
Sure can! Luckily `csvtk` works with gzipped files: [28c23ae](https://github.com/phac-nml/gasnomenclature/pull/29/commits/28c23ae2845999803617ae1b92cd7ee2914c2fe1)
The corresponding test data files were added: [bb91d0d](https://github.com/phac-nml/gasnomenclature/pull/29/commits/bb91d0dd393384273666a826e60549152d61baeb)
I did have to add some syntax to deal with gzipped files when doing a header check with the `append` modules, which can be found here: [3002b00](https://github.com/phac-nml/gasnomenclature/pull/29/commits/3002b00b21dac984ebc1c71d2a886f4cd1974935)